src/pages/workspace/expensifyCard/issueNew/IssueNewCardPage.tsx

```diff
@@ -60,6 +64,18 @@ function IssueNewCardPage({policy, route}: IssueNewCardPageProps) {
         }
     };
 
+    if (isActingAsDelegate) {
+        return (
+            <ScreenWrapper
+                testID={IssueNewCardPage.displayName}
+                includeSafeAreaPaddingBottom={false}
+                shouldEnablePickerAvoiding={false}
+            >
+                <DelegateNoAccessWrapper accessDeniedVariants={[CONST.DELEGATE.DENIED_ACCESS_VARIANTS.DELEGATE]} />
```

Comment:
We should have added a back button press functionality here. Not doing so caused https://github.com/Expensify/App/issues/55720